sdks/go/test/run_integration_tests.sh

```diff
@@ -102,9 +102,8 @@ case $key in
 esac
 done
 
-if [[ "$RUNNER" != "universal" ]]; then
-  PUSH_CONTAINER_TO_GCR='yes'
-else
+PUSH_CONTAINER_TO_GCR='yes'
```

Comment:
Isn't it better to avoid double assignment?

Reply:
Not sure, the root issue seemed to be how the comparison operated though, rather than the assignement. I'll see if I can get the comparison to work properly but otherwise revert this.

Tangent: This is the "preferred" way of handling variable cases like this for Go, where the common case for the variable is used by default, and overridden by the special case. (Obviously if calculating the default value is expensive, and the special case is cheap, the calculus changes, since compute has cost.)

Reply:
I'd guess that is because in Go one wants to use the declare-and-define syntax, which wouldn't work inside the if clause. The convention is opposite in Java/Python (where more value is placed on immutability). As for bash, well, I doubt there's any consistency around there... This is fine if you prefer.

Reply:
A fair observation. I ended up with reversing the if clause anyway since the positive check is clearer.